code/drasil-website/lib/Drasil/Website/About.hs

```diff
@@ -99,6 +105,10 @@ aboutParagraph8 paperGOOL = S "As described in the" +:+ namedRef paperGOOL (S "G
   \generation, allowing Drasil to more efficiently generate code in several languages, including Python, Java, C-Sharp, and C++."
 
 -- | Paragraph providing a link to Drasil papers and documents
-aboutParagraph9 :: Reference -> Sentence
-aboutParagraph9 papersWiki = S "A list of papers and documents written about Drasil can be found in the" +:+ namedRef papersWiki (S "Drasil Papers \
-  \and Documents") +:+ S "wiki page"
\ No newline at end of file
+aboutParagraph9 :: Reference -> Reference -> Reference -> Reference -> Sentence
+aboutParagraph9 papersWiki icsePositionPaper danPoster wellUnderstoodPaper = 
+  S "A list of papers and documents written about Drasil can be found in the" +:+ 
+  foldlList Comma List [namedRef papersWiki (S "Drasil Papers and Documents") +:+. wikiSentence +:+ S "In particular", S "there is a" +:+ 
+  namedRef icsePositionPaper (S "Old Position Paper") +:+ S "outlining our original ideas", S " a" +:+ 
+  namedRef danPoster (S "Drasil Poster"), S " a" +:+ namedRef wellUnderstoodPaper (S "Well-Understood Paper") +:+ 
+  S "discussing key concepts."]
```

Comment:
This is definitely better, but `foldlList` should really just be used for lists of items and not just general cases where a comma is used, for example, something like this (obviously make sure it's correct and the output is right).
```suggestion
  S "A list of papers and documents written about Drasil can be found in the" +:+
  namedRef papersWiki (S "Drasil Papers and Documents") +:+. wikiSentence +:+
  S "In particular" `sC` S "there is" +:+. foldlList Comma List [
    S "an" +:+ namedRef icsePositionPaper (S "Old Position Paper") +:+
      S "outlining our original ideas",
    S "a" +:+ namedRef danPoster (S "Drasil Poster"),
    S "a" +:+ namedRef wellUnderstoodPaper (S "Well-Understood Paper") +:+
      S "discussing key concepts"
  ]
```